tests/e2e/travel.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { test, expect } from '@playwright/test';
+import { TravelPage } from 'e2e-tests/pom/travel.page';
+
+let travelPage : TravelPage;
+
+test.beforeEach('setup', async ({page}) => {
+    travelPage = new TravelPage(page);
+
+  await travelPage.goTo();
+});
+
+test('Validate card number (Visa/Master)', async ({ page }) => {
+await travelPage.loginFlow ('agileway', 'testW1se');
+await travelPage.flightSelect ('New York', 'Sydney', '082025', '15');
+await travelPage.passengerDetails ('Marius', 'Marauskas');
```

Comment:
fillPassengerDetails or enterPassengerDetails will be more correct tai, we should crearly tell which actions we are doing.